core/modules/user/user.admin.inc

```diff
@@ -468,6 +470,14 @@ function user_admin_permissions($form, $form_state, $role_name = NULL) {
       '#markup' => check_plain($role->label),
       '#tree' => TRUE,
     );
+
+    // Amend the entire list of checkboxes with AIRA labels for screen readers.
+    foreach ($all_permissions as $permission_key => $permission_item) {
+      $form['checkboxes'][$role_name][$permission_key]['#attributes']['aria-label'] = t('!role permission: !permission', array(
+        '!role' => $role->label,
+        '!permission' => $permission_item['title'],
```

Comment:
Some permission titles have HTML, and the reader reads this out as well, for example the Files permissions for each file type are in the format "Audio: *Edit own files*" which HTML is: `<em class="placeholder">Audio</em>: Edit own files` and which reads out verbose as "less than slash em class= quote placeholder...". 

Changing to `strip_tags($permission_item['title'])` fixes this.